recipe/meta.yaml

```diff
@@ -37,8 +37,12 @@ outputs:
         - libuv
         - git >={{ git }}
       host:
-        - cudatoolkit {{ cudatoolkit }}        #[build_type == 'cuda']
+{% if build_type == 'cuda' %}
+        - cudatoolkit {{ cudatoolkit }}
+        - openmpi {{ openmpi }} cuda{{ cudatoolkit }}*   #[mpi_type == 'openmpi']
```

Comment:
Might want to add a * before cuda as well in case we eventually add the hash value to the openmpi package.

Reply:
ok - I'll try that out.